### PR TITLE
update for locale fr_FR

### PR DESCRIPTION
**Pull request by forgive38**
some updates

### CODE REVIEW

**Comment by asmecher** [comment]
@pnault, are these changes also intended for the `master` branch, or only `ojs-stable-3_1_0`?

**Comment by pnault** [comment]
@asmecher , I'm pretty sure your comment are not addressed to me but more for @forgive38:
I'm in for fr_CA but fr_FR locales are for the ones that speaks *le français de France* on the other side of the ocean!

**Comment by asmecher** [comment]
Oops, yes, you're correct -- it looks like you were both submitting these at the same time

File: locale/fr_FR/locale.xml [hunk 1]
```diff
@@ -163,7 +163,7 @@
 
 	<!-- Rôle -->
 	<message key="user.role.manager">Directeur de la revue</message>
-	<message key="user.role.subEditor">Rédacteur de rubrique</message>
+	<message key="user.role.sectionEditor">Rédacteur de rubrique</message>
```
Comment (asmecher) [inline locale/fr_FR/locale.xml]: I think this accidentally reverts an intended change.

**Pull request closed without merging**